symengine/CMakeLists.txt

```diff
@@ -270,6 +270,11 @@ if (NOT WITH_SYSTEM_CEREAL)
   set(HEADERS ${HEADERS} ${CEREAL_HEADERS})
 endif()
 
+if (NOT WITH_SYSTEM_FASTFLOAT)
+    file(GLOB_RECURSE FASTFLOAT_HEADERS RELATIVE ${CMAKE_CURRENT_SOURCE_DIR} "utilities/fast_float/include/*")
+    set(HEADERS ${HEADERS} ${FASTFLOAT_HEADERS})
+endif()
+
```

Comment:
Why do we need to install this? They are not exposed by any symengine header right?

Reply:
You're right - we don't need to install this, sorry!